pos_with_tax/static/src/js/main.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * # -*- coding: utf-8 -*-
+ * ##############################################################################
+ * #
+ * #    OpenERP, Open Source Management Solution
+ * #    This module copyright :
+ * #        (c) 2014 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
+ * #                 Antonio Espinosa <antonioea@antiun.com>
+ * #                 Endika Iglesias <endikaig@antiun.com>
+ * #
+ * #    This program is free software: you can redistribute it and/or modify
+ * #    it under the terms of the GNU Affero General Public License as
+ * #    published by the Free Software Foundation, either version 3 of the
+ * #    License, or (at your option) any later version.
+ * #
+ * #    This program is distributed in the hope that it will be useful,
+ * #    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * #    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * #    GNU Affero General Public License for more details.
+ * #
+ * #    You should have received a copy of the GNU Affero General Public License
+ * #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * #
+ * ##############################################################################
+ */
+
+// Check jQuery available
+if (typeof jQuery === 'undefined') { throw new Error('POS with tax Addon requires jQuery') }
+
++function ($) {
+    'use strict';
+
+    openerp.pos_with_tax = function (instance) {
+        var _t = instance.web._t,
+            _lt = instance.web._lt;
+        var QWeb = instance.web.qweb;
+
+        var round_pr = instance.web.round_precision;
+
+        instance.point_of_sale.ProductListWidget.include({
+            compute_all: function(taxes, price_unit) {
```

Comment:
@pedrobaeza, @legalsylvain, we inherit two objects:

* ProductListWidget: where ```compute_all``` and ```get_product_all_prices``` methods do not exist. In this case, OrderLine ```get_all_prices``` can be reused because product selection is done inside method, so I re-implemented the method (this way go against DRY principle, but i didn't find any better way to do) with ```product``` as parameter. Maybe I can re-use ```compute_all```, i'll check it

* Orderline: where I change ```get_display_price``` method. Here super call doesn't apply because I want to change returned value